plugin/src/main/scala/com/softwaremill/clippy/AdviceLoader.scala

```diff
@@ -10,12 +10,25 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.io.Source
 import scala.tools.nsc.Global
 import scala.util.{Failure, Success, Try}
+import scala.collection.JavaConversions._
 
 class AdviceLoader(global: Global, url: String, localStoreDir: File, projectAdviceFile: Option[File])(implicit ec: ExecutionContext) {
   private val OneDayMillis = 1000L * 60 * 60 * 24
 
   private val localStore = new File(localStoreDir, "clippy.json.gz")
 
+  private val resourcesAdvice: List[Advice] = {
+    getClass.getClassLoader
+      .getResources("clippy.json")
+      .toIterator
+      .map{ url =>
+        val is = url.openStream()
+        val clips = inputStreamToClippy(is)
+        is.close()
+        clips.advices
+      }
+  }.flatten.toList
```

Comment:
I would add some error-handling here. E.g. wrap reading advice from each URL in a `Try`, and then collect only the successful ones, discarding (logging a warning) the ones that failed. That way if a single library would contain malform `clippy.json` file, that wouldn't break the whole compilation process.